scenario/changesheet.go

```diff
@@ -21,7 +22,7 @@ import (
 type (
 	// ChangeSheetSettings settings for change sheet action
 	ChangeSheetSettings struct {
-		ID string `json:"id" displayname:"Sheet ID" doc-key:"changesheet.id"`
+		ID string `json:"id" displayname:"Sheet ID" doc-key:"changesheet.id" appstructure:"sheet"`
```

Comment:
Ok, understand why you want this here now, to filter out only this type. In that case it's fine, but the filtered type shouldn't be hardcoded as it's own parameter in AppStructurePopulatedObjects, it needs to be more generic so any object type can be filtered, e.g. `pivot-table` or any custom object type. Filter applied on object dictionary using visualization. Question is, how do we separate between applying filter on active objects or all objects?

Reply:
It's kinda hard coded, but kinda not. From an API perspective, you're free to specify any object type, including pivot-table etc. In that case it will filter from the object map automatically.

There's no support for non-active objects.

Reply:
Since we know subscribeobject is upcoming within the coming days, there's no need to implement it in a way that we need to throw away almost directly.

Reply:
I still don't follow. Subscribeobject won't affect this implementation. Changesheet will still only change sheets, not other types of objects.

Reply:
The only thing that appstructure:"sheet" does is tell the GUI that the changesheetsettings.ID should have a dropdown with all the active sheets in it, which won't change? Even if I merge the sheet and object lists as suggested, it doesn't have any bearing on this API.

Reply:
I still don't see why this is hardcoded, What if I implement my own type of container, Then make an activate container action. I would need the same kind of logic, doesn't make sense to hardcode it.